src/broker/api/mod.rs

```diff
@@ -1 +1,2 @@
-pub mod receiver;
```

Comment:
może to wrzucić do oddzielnego folderu? Zacząłem rzeźbić u siebie i przemianowałem api -> receiver. To tak żeby nie mieć worka "api" na wszystko, tylko foldery/moduły dla każdej domeny/grubszej funkcjonalności